### PR TITLE
Add Tails to IPC

### PR DESCRIPTION
**Pull request by Suraru**
From floofstation, original PR by Fasana

https://github.com/Fansana/floofstation1/pull/253

### CODE REVIEW

**Comment by Aidenkrz** [comment]
This doesn't make sense to me.

File: Resources/Prototypes/Voice/Voice - Shortcut.lnk
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Voice/Voice - Shortcut.lnk:1]: what is this?

**Comment by Suraru** [comment]
This adds tails to the markings section for IPCs. So you can basically have a Kerfur

**Comment by Aidenkrz** [comment]
I don't think IPCs should have organic tails, if some cool ones were designed for them I'd think that would work but as it stands I don't like this.

**Comment by DangerRevolution** [comment]
Why violate the core design principle of an autonomous android and have a Kerfur (picture attached); when you could have just the autonomous android (with a good and unique design); or add robotic markings to Felinids (as many people have requested)?

**Comment by Ichaie** [comment]
This has already been done in Estação Pirata, and from the experience and feedback of some players, it is not such a good idea, if you are going to add "organic" tails it will not make much sense, I believe that adding tails for example to vulps and felinids will end up taking away a unique characteristic of a playable species, but it might be cool to add original models for IPC

**Comment by Aidenkrz** [comment]
> Why violate the core design principle of an autonomous android and have a Kerfur (picture attached); when you could have just the autonomous android (with a good and unique design); or add robotic markings to Felinids (as many people have requested)?
> 
> 

kerfur

**Comment by DEATHB4DEFEAT** [comment]
I think this is a good idea, but organic tails on them are weird. You can add tails meant for robots if you want.

**Pull request closed without merging**